sdk/textanalytics/Azure.AI.TextAnalytics.Legacy.Shared/src/Custom/JobManifestTasks.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics.Legacy
+{
```

Comment:
I absorbed the content for things in the `Custom` area from @nisha-bhatia's work.  I'm not entirely sure that we need the extra code; I was able to get the legacy bits to generate and compile without it, but I left it for now under the assumption that we'll need it as we start building out the service clients.   If that turns out to be incorrect, I'll look at trimming things back then.

Reply:
Yup; that's the scenario.   Nothing public, just overrides and extensions for generated types that were in the `Internal` folder.